src/RA_Defs.cpp

```diff
@@ -10,68 +10,150 @@ GetParseErrorFunc GetJSONParseErrorStr = GetParseError_En;
 #endif
 
 static_assert(sizeof(BYTE*) == sizeof(char*), "dangerous cast ahead");
+
+_Use_decl_annotations_
 char* DataStreamAsString(DataStream& stream)
 {
     return reinterpret_cast<char*>(stream.data());
 }
 
+_Use_decl_annotations_
 std::string Narrow(const wchar_t* wstr)
 {
-    static std::wstring_convert< std::codecvt_utf8_utf16< wchar_t >, wchar_t > converter;
+    std::wstring_convert< std::codecvt_utf8_utf16< wchar_t >, wchar_t > converter;
     return converter.to_bytes(wstr);
```

Comment:
Having these static made no sense, RichPresense stuff still worked. The only thing I can think of that should be static is a mutex or semaphore, and that's about it or anything that deals with synchronization